Source/WebCore/platform/graphics/cairo/CairoUtilities.cpp

```diff
@@ -49,13 +49,17 @@
 
 namespace WebCore {
 
-#if USE(CAIRO) && !PLATFORM(GTK)
+static auto s_defaultCairoFontOptions = CairoUniquePtr<cairo_font_options_t>(cairo_font_options_create());
+
 const cairo_font_options_t* getDefaultCairoFontOptions()
 {
-    static NeverDestroyed<cairo_font_options_t*> options = cairo_font_options_create();
-    return options;
+    return s_defaultCairoFontOptions.get();
+}
+
+void setDefaultCairoFontOptions(cairo_font_options_t* options)
```

Comment:
This should receive a `CairoUniquePtr<cairo_font_options_t>&&` to make it clear we are transferring the ownership.